méli-mélo/2024-04-stepsquiz/reports/a11y-1-en.html

```diff
@@ -0,0 +1,94 @@
+description: "Accessibility assessment for the progress bar element."
+lang: en
+---
+<p>Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2</p>
+
+<h2>GCWeb implementation plan</h2>
+
+{% assign implPlan = site.pages | where: "output", "false" | where: "componentName", "2024-04-stepsquiz" | first %}
+
+<nav>
+	<h3>On this page</h3>
+	<ul>
+		<li><a href="#summary">Summary</a></li>
+		<li><a href="#acr">Assessement details</a></li>
+	</ul>
+</nav>
+
+<section>
+	<h3 id="summary">Summary</h2>
+	<p>This is an informal assessment based on an accessibility testing rule set. It cannot be seen as an accessibility conformance report. The latter is published separately from the assessment and only indicates if the requirements are satisfied, not satisfied, or if further testing is needed as specified by establishing the conformity described in WCAG 2.1. This assessment can contain false positives, false negatives, or be accurate. The assessment interpretation, via the accessibility conformance report, can consider other independent assessments in order to establish the conformance.</p>
+
+	<dl >
+		<dt>Type:</dt>
+		<dd>Accessibility assessment</dd>
+		<dt>Asserted by:</dt>
+		<dd><a href="https://esdc.prv/en/hrsb/accessibility/roadmap.shtml">ESDC Accessibility Office</a></dd>
+		<dt>Date:</dt>
+		<dd>2024-06-25</dd>
+		<dt>Subject:</dt>
+		<dd>
+			<ul class="list-unstyled">
+				<li><strong>Page / Screen title</strong>: Questionnaire - What to do when someone dies</li>
+				<li><strong>Page URL</strong>: <a href="https://www.canada.ca/en/employment-social-development/employment-social-development/navigating-death/questionnaire.html">https://www.canada.ca/en/employment-social-development/employment-social-development/navigating-death/questionnaire.html</a></li>
+				<li><strong>Applicability:</strong> In the context of the life events project conduct by Canada.ca Journey Lab.</li>
+				<li><strong>HTML element:</strong> <code>&lt;progress&gt;</code></li>
+			</ul>
+		</dd>
+		<dt>Toward a conformity at:</dt>
+		<dd>WCAG 2.1 (Option: Level AA)</dd>
+		<dt>Summary:</dt>
+		<dd>Partial evaluation focused only on the component not its container. Evaluated all Success Criteria at Level AA of WCAG 2.2</dd>
+	</dl>
+
+	<h3 id="acr">Assessment details</h3>
+	<table class="table table-hover table-bordered" cellspacing="0" rules="all" border="1" id="MainContent_FormView_GridViewPoints" style="border-collapse:collapse;">
+		<tbody
+			<tr>
+				<th scope="col">Standard Name</th><th scope="col">Point Name</th>
+			</tr>
+			<tr>
+				<td>Web Content Accessibility Guidelines</td><td>1.3.1 Info and Relationships (Level A)</td>
+			</tr>
+			<tr>
+				<td>Web Content Accessibility Guidelines</td><td>3.3.2 Labels or Instructions (Level A)</td>
+			</tr>
+			<tr>
+				<td>Web Content Accessibility Guidelines</td><td>4.1.2 Name, Role, Value (Level A)</td>
+			</tr>
+		</tbody>
+	</table>
+	<dl class="dl-horizontal">
```

Comment:
Let add a sub heading for readability
```suggestion
	<h4>Error 1 - Form elements must have labels</h4>
	<dl class="dl-horizontal">
```